example/src/main/java/com/mattskala/itemadapter/example/MainActivity.kt

```diff
@@ -16,15 +18,19 @@ class MainActivity : AppCompatActivity() {
 
     private val adapter = ItemAdapter()
 
+    private val items = createItems()
+    private val selectedItemsTitle = mutableListOf<String>()
```

Comment:
It would be better to store the selected item IDs, as the titles can be duplicated in "What Next?" items. However, do we need to store selected items separately at all since the selected state is already a part of the Item object? The selected state is duplicated and there is no clear source of truth.

Reply:
That's true, I was just doing this very quickly to test if it works. I'll change it